dev/com.ibm.ws.jaxws.common/bnd.bnd

```diff
@@ -49,8 +57,8 @@ Import-Package: \
    *
 
 Private-Package:\
-   com.ibm.ws.jaxws.*, \
-   com.ibm.wsspi.jaxws
+   com.ibm.ws.jaxws.*;version=${exportVer},\
```

Comment:
As before.  Not sure why version is needed in private package

Reply:
removed